test/package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "test": "node --preserve-symlinks test.js test",
     "coverage": "cross-env NODE_ENV=test nyc --reporter=lcov --reporter=text-summary npm test",
-    "coverage-ci": "cross-env NODE_ENV=test nyc --reporter=lcovonly npm test",
+    "coverage-ci": "cross-env NODE_ENV=test nyc --reporter=text-lcov npm test > coverage.lcov",
```

Comment:
Is this change required?  I think we may be piping like this elsewhere, but I think there are also examples of a command line option that does not require special pipes.  Might also involve changing the path to the coverage report on L74 of the workflow.

Reply:
Thanks, that change was not required. I've reverted it back and updated the coverage file path.